subprojects.json

```diff
@@ -0,0 +1,548 @@
+[
```

Comment:
Is this artifact used?

Reply:
derp don't know how this got checked in, thanks